xml/ns-Microsoft.VisualC.StlClr.Generic.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.VisualC.StlClr.Generic">
   <Docs>
-    <summary>Contains the classes, interfaces, and iterators that are used to implement the generic interface to the . By using this generic interface, other .NET languages, such as C# and Visual Basic, can call code that is written using STL/CLR.</summary>
+    <summary>Contains the classes, interfaces, and iterators that are used to implement the generic interface to the STL/CLR. By using this generic interface, other .NET languages, such as C# and Visual Basic, can call code that is written using STL/CLR.</summary>
```

Comment:
```suggestion
    <summary>Contains the classes, interfaces, and iterators that are used to implement the generic interface to the STL/CLR Library. By using this generic interface, other .NET languages, such as C# and Visual Basic, can call code that is written using STL/CLR.</summary>
```
Here is a previous version of the description (from <https://docs.microsoft.com/en-us/previous-versions/gg145040(v%3dvs.110)>:

![image](https://user-images.githubusercontent.com/10886961/66143579-407c2180-e5bc-11e9-9bb8-81a376588bb3.png)

Reply:
I've addressed your feedback @rpetrusha and also restored the link that once existed as well:
https://msdn.microsoft.com/en-us/library/microsoft.visualc.stlclr.generic(v=vs.100)?toc=xxx